src/SimulationRunner.cc

```diff
@@ -237,6 +237,12 @@ SimulationRunner::SimulationRunner(const sdf::World *_world,
 
   ignmsg << "Serving world SDF generation service on [" << opts.NameSpace()
          << "/" << genWorldSdfService << "]" << std::endl;
+
+  std::string entitySystemService{"entity/system/add"};
```

Comment:
that works. I've moved the logic to SystemManager and as the result the service now does not have the `/world/<world_name>/` prefix

Reply:
Ah I didn't realize that would be a consequence of the move. Is there a way we can propagate the node name to `SystemManager`? In theory there could be entities with the same ID on different worlds. We don't really support that use case right now but it would be good to keep following the existing pattern so we don't prevent that use case in the future.

Reply:
ok since the SystemManager.hh is not installed. I modified it to take one extra `_namespace` arg in the constructor. The service names now have the `/world/<world_name>` namespace. [8335ebd](https://github.com/gazebosim/gz-sim/pull/1524/commits/8335ebdb30381e4f6e04f0ca323b83b771168fe8)